src/Pages/MapPage/MapFilter.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import CheckBox from './MapFilterComponent/CheckBox';
+import Slider from './MapFilterComponent/Slider';
+
+export default function MapFilter({
+  setMaintenanceCost,
+  setExclusiveM2,
+  filterBox,
+  setFilterBox,
+}) {
+  const [currentSubFilter, setCurrentSubFilter] = useState(0);
+
+  const filterList = [
+    ['원룸, 투 쓰리룸, 오피스텔', 'room'],
```

Comment:
```suggestion
  const filterList = [
    { text: '원룸, 투 쓰리룸, 오피스텔', type: 'room'},
```
이 경우에는 배열보다 객체가 데이터에 부합하는 데이터 형태로 보입니다